env.example

```diff
@@ -1,8 +1,12 @@
 # use unique prefix for Azure resources when it is set, otherwise use your user's name
 export AZURE_PREFIX="${AZURE_PREFIX:-$USER}"
-export LOCATION=eastus
-export ARO_IMAGE=arointsvc.azurecr.io/aro:latest
+export LOCATION=westeurope
 export NO_CACHE=false
 export AZURE_EXTENSION_DEV_SOURCES="$(pwd)/python"
 
-. secrets/env
+export CLUSTER_RESOURCEGROUP="${USER}-v4-$LOCATION"
+export CLUSTER_NAME="${USER}-aro-cluster"
+export CLUSTER_VNET="${USER}-aro-vnet"
+ARO_IMAGE=arointsvc.azurecr.io/aro:latest 
```

Comment:
Nit: this probably needs to be `export`-ed.

Reply:
Thanks for pointing that out! The env file is now updated.